layouts/_default/author.terms.html

```diff
@@ -1,60 +1,82 @@
 {{ define "extra-main" }}
   {{ $pg := partial "helper/paginator" . }}
   {{ $isoFormat := "2006-01-02T15:04:05Z07:00" }}
-  <section class="section columns" data-ga-category="authors">
-    <div class="column is-three-fifths is-offset-one-fifth">
-      {{ range $pg.Pages.ByTitle }}
-        {{ $author := partial "helper/get-author" .Title }}
-        <div class="level">
-          <div class="level-left">
-            <div class="level-item">
-              {{ if $author.thumb }}
-                <a href="{{ $author.link }}" class="image is-64x64">
-                  <img
-                    src="{{ $author.thumb }}"
-                    alt="Photo of {{ $author.name }}"
-                    class="is-rounded is-monochrome"
-                  />
-                </a>
-              {{ else }}
-                <span class="image is-64x64"></span>
-              {{ end }}
-            </div>
+  <section class="bg-white py-10 px-2">
+    <div class="mx-auto w-full max-w-prose text-lg">
+      <ul
+        class="mx-auto my-14 mt-10 w-full"
+        data-ga-category="inline-footer:author"
+      >
+        {{ range $pg.Pages.ByTitle }}
+          {{ $author := partial "helper/get-author" .Title }}
+          <li
+            id="{{ $author.slug }}"
+            class="flex items-center justify-start space-x-4 mb-4"
+          >
+            {{ if $author.headshot }}
+              {{ $sizes := (slice
+                (dict "width" 64 "height" 64)
+                (dict "width" 64 "height" 64)
```

Comment:
I think this was supposed to be 64x64 then 128x128. It's blurry on my retina MacBook:

<img width="430" alt="image" src="https://user-images.githubusercontent.com/222245/227003654-0a06e049-12b6-46ec-87ee-ee3f674160e0.png">